NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget.orgFeed" value="https://identitydivision.pkgs.visualstudio.com/IDDP/_packaging/nuget.orgFeed/nuget/v3/index.json" />
```

Comment:
Created a feed with nuget.org and upstream feed for internal analyzers: https://identitydivision.visualstudio.com/IDDP/_packaging?_a=feed&feed=nuget.orgFeed. This is one way I found to create a feed to include in nuget.config where we can have public and private sources.